setup.py

```diff
@@ -65,6 +65,7 @@
     'invenio-celery>=1.1.1',
     'invenio-rest[cors]>=1.1.2',
     'simplejson>=3.0.0',
+    'six>=1.12.0',
```

Comment:
now that we are on Python 3, we should not need six anymore right?

Reply:
Since this module was there before we keep support for Python 2.x for a bit (AFAIK).

Reply:
ah ok!

Reply:
*I already gave my approval and should have provided those yesterday, so feel free to ignore my comments. I try to be consistent and want to get those remarks out there*: 

if we are keeping support for python 2 for a bit, we should keep testing against it in Travis. We should only remove the Python 2 tests, when we remove the support for it I think.

Reply:
This module (invenio-files-rest) is already released (is not like RDM ones) so this one keeps support for Python 2 (as you can see in travis, it keeps testing those: https://github.com/inveniosoftware/invenio-files-rest/blob/master/.travis.yml#L57)

The new RDM modules we are developing are alpha/not stable yet, and since no one should be relying on them until June we only need to support Python 3.

Reply:
My point was that Travis testing of Python 2 is removed *in this PR* . See at the top...

Reply:
Yeah, you are right. Mistake, talked to Lars. We do not "test" it anymore (e.g. if it would break we wouldn't fix it) however removing `six` has more implications since it is used widely across the code base of the module (and it will work anyway with Python 3).

Removing it will be a separate issue/task at some point.

Reply:
I see... I was further confused by the fact that `six` is *added* here, although it's already used... I made the issue for switching to Python 3 completely: https://github.com/inveniosoftware/invenio-files-rest/issues/239 .